gradle/verification-metadata.xml

```diff
@@ -1615,6 +1615,11 @@
             <sha256 value="24150c7dbdc8a5bb987cbeb4898ee1515d2893d3f6224acfc563da1a7fd5c283" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="io.fabric8" name="kubernetes-client-bom" version="5.12.2">
```

Comment:
I'm surprised at all the changes in this file - are they are relevant for this PR?

Reply:
yes, that surprised me too. Those are added by `org.apache.logging.log4j:log4j:2.20.0` 😞 ([here](https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j/2.20.0/log4j-2.20.0.pom)'s the `log4j-2.20.0.pom`)